README.md

```diff
@@ -31,13 +31,16 @@ jobs:
         path: ~/.cache/pre-commit
         key: pre-commit|${{ env.PY }}|${{ hashFiles('.pre-commit-config.yaml') }}
     - uses: pre-commit/action@v1.0.1
+      with:
+        extra_args: ['flake8', '--all-files']
```

Comment:
this shouldn't be part of the default template -- I want this block to be what people copy paste for the right thing 99% of the time

the new setting should be documented separately (it should be rare that it is needed / used)

Reply:
Will do.

Reply:
@asottile Sorry for the delay; this is done!